### PR TITLE
divide status values by 10, change message definitions

### LINKED ISSUE #102: Check for errors periodically
We have a method `errorCheck()` (for which the docstring also needs to be updated as it's incorrect, and should be renamed `error_check()`) that is only run once on start-up. Often, if there are errors, they only show up when the motor moves or when a command is sent. We should check periodically and raise a flag when an error occurs.

### PR DESCRIPTION
**Pull request by Achllle**
See #97 and #58 

also closes #102 

This also checks for errors more continuously (this previously only happened on startup) and logs them to console in addition to adding them to the message.

### CODE REVIEW

**Comment by gfosmire** [comment]
@Achllle Temp and Voltage look good, but according to the roboclaw manual on page 68 you need to divide by 100 to get to Amps.

http://downloads.basicmicro.com/docs/roboclaw_user_manual.pdf

The other changes to the naming and how errors are handled seem good too

**Comment by Achllle** [comment]
Hah, good catch! I swear it was documented differently elsewhere...

File: ROS/osr/src/roboclaw_wrapper.py [hunk 1]
```diff
@@ -386,46 +385,50 @@ def velocity2qpps(self, velocity, ticks_per_rev, gear_ratio):
...
-    def getCurrents(self):
+    def read_currents(self):
         currents = [None] * 10
         for i in range(5):
             currs = self.rc.ReadCurrents(self.address[i])
-            currents[2*i] = currs[1]
-            currents[(2*i) + 1] = currs[2]
+            # reported by roboclaw in 10ths of an Ampere
+            currents[2*i] = currs[1] / 10.0
+            currents[(2*i) + 1] = currs[2] / 10.0
```
Comment (Achllle) [inline ROS/osr/src/roboclaw_wrapper.py:407]:
```suggestion
            currents[2*i] = currs[1] / 100.0
            currents[(2*i) + 1] = currs[2] / 100.0
```

Comment (Achllle) [reply ROS/osr/src/roboclaw_wrapper.py:407]: @gfosmire just an FYI this is a cool feature of GitHub where you can submit suggestions to code so the PR opener can merge in those changes through GitHub. Select a couple of lines in review mode and hit the left-most icon 'Suggestion' and when you make changes, it'll show up like the above

**Pull request merged**